app/views/users/show.html.erb

```diff
@@ -12,10 +12,20 @@
 
 </section>
 
+<% if current_user.skills.size > 0 %>
 <section class="my-skills">
-
+  <h3>My Skills:</h3>
+  <ul>
+    <% current_user.skills.each do |skill| %>
+    <li><%= skill.skill %></li>
```

Comment:
skill.skill is....weird looking

Reply:
but fine?

Reply:
might change the first skill to be `user_skill` for readability

Reply:
that would work